### PR TITLE
trac#501: Do not print tooltips in printed form of HTML report

### PR DESCRIPTION
**Pull request by jan-cerny**
Tooltips will not be printed because it was repeating redundant information.
It caused lines to span to a half of page.

### CODE REVIEW

**Comment by jan-cerny** [comment]
If this will be approved, it also requires to push here updated xccdf-resources.xsl

File: xsl/xccdf-resources/openscap.css [hunk 1]
```diff
@@ -40,3 +40,10 @@ div.top-spacer-10
 {
     margin-top: 10px;
 }
+
+@media print
+{
+    .container { width: 100% }
```
Comment (mpreisler) [inline xsl/xccdf-resources/openscap.css]: what is the reason behind the `width: 100%` rule?

Comment (jan-cerny) [inline xsl/xccdf-resources/openscap.css]: It is to do not have very big unused white edges of paper on left and on right.

**Comment by mpreisler** [comment]
`::after` is the correct standard CSS syntax, `:after` is done for compatibility with IE8, would be great to have a short comment about that in the CSS.

**Comment by jan-cerny** [comment]
Is it really needed to write there comments like that? Is there anything that blocks merging this?

**Comment by mpreisler** [comment]
It is not strictly necessary but it is important. When the next person glances over the CSS they want to know why we are using invalid syntax.

**Comment by jan-cerny** [comment]
Reworked - less generic selectors + a comment

**Comment by mpreisler** [comment]
Thx

**Pull request merged**